packages/keystone/bin/commands/create-tables.js

```diff
@@ -0,0 +1,56 @@
+const path = require('path');
+const chalk = require('chalk');
+const { DEFAULT_ENTRY } = require('../../constants');
+const { getEntryFileFullPath } = require('../utils');
+const { asyncForEach } = require('@keystonejs/utils');
+const createTables = async (args, entryFile, spinner) => {
+  // Allow the spinner time to flush its output to the console.
+  await new Promise(resolve => setTimeout(resolve, 100));
+  const { keystone } = require(path.resolve(entryFile));
+  await keystone.connect(); // Need to do _createTables post connect so awaiting connect
+  keystone._consolidateRelationships();
+  let errors = false;
+  await asyncForEach(Object.values(keystone.adapters), async adapter => {
+    if (!adapter._createTables) {
+      spinner.info(chalk.yellow.bold(`create-tables is only required for KnexAdapter`));
+      return null;
+    }
+    try {
+      console.log('Creating tables...');
+      await adapter._createTables();
+    } catch (e) {
```

Comment:
/nit -- Bob Martin insists that we call this `error` 😋

Reply:
I don't know or care who Bob is